src/common/helpers.js

```diff
@@ -0,0 +1,4 @@
+export const kelvinToCelsius = kelvin => {
+  const celsius = kelvin - 273;
+  return +celsius.toFixed(1);
```

Comment:
Try to avoid using non-obvious ways of type casting. Use Number constructor instead, or parseInt